server/controllers/UsersController.js

```diff
@@ -269,6 +269,46 @@ class UsersController {
         next(err);
       });
   }
+
+  /**
+   * @description - This method gets a single user profile
+   * @param {object} req The express request object
+   * @param {object} res The express response object
+   * @param {object} next The express next object
+   * @returns {void}
+   */
+  static async getSingleProfile(req, res, next) {
+    // requester id
+    const requesterId = req.userData.id;
+    // profile id
+    const { userId } = req.params;
+
+    let userProfile;
+    try {
+      // get the single user by id
+      userProfile = await User.findByPk(userId, {
+        raw: true,
+      });
+    } catch (err) {
+      next(err);
+    }
+
+    // check if the requester is not the owner of the user profile
+    if (requesterId !== +userId) {
+      // delete sensitive information
+      delete userProfile.email;
+      delete userProfile.password;
```

Comment:
All of these except `email` and `updatedAt` should not be shown to the user. What will he use his fbId, twitterId for?

Reply:
What will the user do with his password on the frontend? What will you need the fb, twitter or google IDs for on the front-end? @Darthrighteous

Reply:
Yeah the password field has even been removed with the new implementation we adopted. I'll make these updates.

Reply:
@mbilesanmi fixed!